chapter_linear-networks/linear-regression-scratch_vn.md

```diff
@@ -197,10 +198,10 @@ For example, it requires that we load all data in memory and that we perform lot
 The built-in iterators implemented in Apache MXNet are considerably efficient and they can deal both with data stored on file and data fed via a data stream.
 -->
 
-Khi chạy bộ duyệt, ta lấy từng minibatch cho đến đến khi đã lấy hết bộ dữ liệu.
-Mặc dù sử dụng bộ duyệt như trên phục vụ tốt cho công tác giảng dạy, nó lại không phải là cách hiệu quả và có thể khiến chúng ta gặp nhiều rắc rối trong thực tế.
-Ví dụ, nó buộc ta phải nạp toàn bộ dữ liệu vào bộ nhớ, do đó phải thực thi rất nhiều thao tác truy cập bộ nhớ ngẫu nhiên. 
-Các bộ duyệt trong Apache MXNet lại khá hiệu quả khi chúng có thể xử lý cả dữ liệu lưu trữ trên tập tin lẫn các luồng dữ liệu. 
+Khi chạy iterator, ta lấy từng minibatch cho đến khi lấy hết bộ dữ liệu.
```

Comment:
```suggestion
Khi chạy iterator, ta lấy từng minibatch cho đến khi lấy hết bộ dữ liệu.
```
Đoạn này dịch thiếu từ "distinct", vì trên lý thuyết có thể lấy minibatch có dữ liệu giống nhau được.  
Dịch thiếu từ trong ngoặc (try this)